torch/testing/_internal/common_nn.py

```diff
@@ -4662,8 +4662,9 @@ def __init__(self, *args, **kwargs):
         self.skip_double = kwargs.get('skip_double', False)
 
     def _do_test(self, test_case, module, input):
+        num_threads = torch.get_num_threads()
+        torch.set_num_threads(1)
```

Comment:
Do we have a context manager to do this by any chance?

Reply:
I did not find it, but I did not look hard. Also did not make sense to create a context manager just for this.

Reply:
My concern was more that if the test fails, it will keep the setting for all other tests and could cause other tests to falsely fail as well (unlikely but still).
I would agree that a context manager just for test might be overkill.